### PR TITLE
Nazar/feature/plants screen

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: src/components/Themed.tsx
Comment (Taraslysak) [inline src/components/Themed.tsx:1]: THIS FILE SHOULD BE DELETED!!!!!!!!!!!!!!!!!!!!!!!

File: src/screens/Plants/Plants.tsx [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
 import { Text, View } from '@src/components/Themed';
 import { styleSheet } from './Plant.style';
+import { Categories } from '@src/components/Categories/Categories';
+import { useState } from 'react';
 
 export default function PlantsScreen() {
 	const { styles } = useStyles(styleSheet);
+	const [categoryUuids, setCategoryUuids] = useState<string[]>([]);
+
+	const setUuid = (uuid: string) => {
```
Comment (Taraslysak) [inline src/screens/Plants/Plants.tsx:11]: `setUuid` => `handleSelectCategory`?

File: src/components/Categories/Categories.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			queryKey: [queryKeys.GET_PLANT_CATEGORIES],
+		},
+	});
+	const categories = data?.data ?? [];
+
+	return (
+		<View style={styles.wrapper}>
+			<Text style={styles.title}>{t('components.categories')}</Text>
+			<ScrollView horizontal={true} showsHorizontalScrollIndicator={false} contentContainerStyle={styles.categories}>
+				{categories.map((category) => (
```
Comment (Taraslysak) [inline src/components/Categories/Categories.tsx:31]: {!!data?.data && data?.data.map((category) => (

**Review by Taraslysak — CHANGES_REQUESTED** [review]
Please, remove Themed file from the project, since it is redundant

**Review by nixissmtp — APPROVED** [review]

**Pull request merged**